index.js

```diff
@@ -32,32 +34,42 @@ async function checkChangelog() {
   if(lerna) {
     const lernaPackages = JSON.parse(await execAndReturnOutput(`npx lerna changed --json --loglevel silent`));
     var errors = "";
-    var changedLocal = false;
-    var headerFoundLocal = false;
     for (const package of lernaPackages) {
-      const fileLocation = path.relative(directory, package.location);
+      const changelogLocation = path.join(path.relative(directory, package.location), file);
+      let changedLocal = false;
+      let headerFoundLocal = false;
+      let found = false;
```

Comment:
Is `found` a redundant variable? It's value always seems identical to `changedLocal`.

Reply:
Yep, that seems to be the case.